src/main/webapp/script.js

```diff
@@ -46,40 +49,30 @@ function cleanseUrl(url) {
  * Fetches data and adds to html
  */
 function getChart() {
+  $('form').submit(function() {
     const data = new google.visualization.DataTable();
     data.addColumn('string', 'Range');
     data.addColumn('number', 'Count');
-    // TODO: change hardcoded data to sentiment scores (once that part is finished)
-    data.addRows([
-        ['-1.0', null],
-        [null, 1],
-        ['-0.8', null],
-        [null, 1],
-        ['-0.6', null],
-        [null, 4],
-        ['-0.4', null],
-        [null, 5],
-        ['-0.2', null],
-        [null, 10],
-        ['0.0', null],
-        [null, 10],
-        ['0.2', null],
-        [null, 10],
-        ['0.4', null],
-        [null, 10],
-        ['0.6', null],
-        [null, 15],
-        ['0.8', null],
-        [null, 15],
-        ['1.0', null]
-    ]);
+
+    currentLabel = LOWEST_SCORE;
+
+    while (currentLabel < HIGHEST_SCORE) {
```

Comment:
You can simply write a for loop here instead of incrementing below:
for (CurrentLabel = LOWEST_SCORE; CurrentLabel < HIGHEST_SCORE; CurrentLabel += 0.2) {}

This way you can get rid of the currentLabel outside the for loop as well.